### DISCUSSION TITLE
Run Test Error

### DISCUSSION THREAD

**Discussion by mohamad-elbohsaly** [artilleryio/artillery#1302] [Artillery]
I have in my folder a CSV and a YML file containing artillery configurations. I am facing an error while running the artillery command as you can see below
<img width="1234" alt="Screen Shot 2022-01-22 at 1 41 55 PM" src="https://user-images.githubusercontent.com/18421260/150637139-af103f63-8ec7-4d5b-93a8-b393afd15929.png">
:

**Comment by hassy** [comment]
hi @mado-bohsali 👋 most likely source of the error is that `scenarios` section is undefined (possibly an indentation issue?) - https://www.artillery.io/docs/guides/guides/test-script-reference#scenarios-section

**Reply by mohamad-elbohsaly** [reply]
> hi @mado-bohsali 👋 most likely source of the error is that `scenarios` section is undefined (possibly an indentation issue?) - https://www.artillery.io/docs/guides/guides/test-script-reference#scenarios-section

The problem was an indentation problem. No worries.
